src/CounterInterceptor/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿namespace TemporalioSamples.CounterInterceptor;
+
+using Temporalio.Client;
+using Temporalio.Worker;
+
+internal class Program
+{
+    private static async Task Main(string[] args)
+    {
+        static string GetEnvVarWithDefault(string envName, string defaultValue)
+        {
+            string? value = Environment.GetEnvironmentVariable(envName);
+            if (string.IsNullOrEmpty(value))
+            {
+                return defaultValue;
+            }
+            return value;
+        }
+
+        var address = GetEnvVarWithDefault("TEMPORAL_ADDRESS", "127.0.0.1:7233");
+        var temporalNamespace = GetEnvVarWithDefault("TEMPORAL_NAMESPACE", "default");
+        var tlsCertPath = GetEnvVarWithDefault("TEMPORAL_TLS_CERT", string.Empty);
+        var tlsKeyPath = GetEnvVarWithDefault("TEMPORAL_TLS_KEY", string.Empty);

```

Comment:
This is not something we do with our other samples. If/when we come back and make samples work across many environments, we will update them all. No need to have just this sample accept environment variables unlike others.

Reply:
I get what you are saying and really wish we did this everywhere...

Reply:
We plan to